ci-integration/run-test-docker.sh

```diff
@@ -42,10 +44,12 @@ ignoredirs=(services/core/DNP3Agent services/core/IEEE2030_5Agent services/core/
 # failed.
 HAS_FAILED=0
 
+set +e # allow this script to handle errors.
+
```

Comment:
The logs get filled with a lot of the commands from the helper methods in this script. Is the signal worth the noise? Would allowing the option to uncomment a `set +x` to remove the logging of shell commands be a good idea?

Reply:
I think the idea was to have as much logging on the actual process itself.  I bet we can get rid of the +e after we have made sure that the test-docker script works properly.